### ISSUE TITLE
Fix Commcare integration

### ISSUE THREAD

**Issue by mahmoodkhan** [toladata/TolaTables#433]
### Current behavior
When importing from Commcare, data is imported only at the project level. 

### Expected behavior
Users should be able to select a specific form or case to import data from.
Also rename the menu item from "Import from CommCare case data" to "Import from CommCare"

### Steps to reproduce the behavior

1. Click on Import Data > Import from CommCare case data
2. It does not allow you to choose a specific dataset, it just starts importing at the project level.

### Additional details
 Commcare limits API access to users with a pro-level subscription. It is not available to free users.

**Comment by jefmoura** [comment]
@mahmoodkhan It has been removed since we're not using it. https://github.com/toladata/TolaTables/issues/428

**Comment by mahmoodkhan** [comment]
@jefmoura perhaps we can make it conditional so that it only shows up for Mercy Corps? or do you suggest we switch to the toladata-foundation and implement it there?

**Comment by jefmoura** [comment]
@mahmoodkhan if somebody fixes the integration and it works well, we can keep it

**Comment by mahmoodkhan** [comment]
@jefmoura definitely, will make sure it's fully functional; however, as noted in the issue's description, CommCare APIs are only available for paid (pro) subscribers. It will not work with CommCare free plans

**Comment by sanjuroj** [comment]
@jefmoura, @mahmoodkhan Just to clarify, when I run the CommCare download on my laptop using the dev-v2 branch, it completes successfully.  Is it possible that the problem is related to server setup issues rather than the code?  

The work we're planning to do will make the downloaded data more useful, but that won't fix the any current issues with the implementation.
